src/fernet/core.clj

```diff
@@ -6,6 +6,8 @@
            (javax.crypto.spec IvParameterSpec SecretKeySpec)
            (org.bouncycastle.jce.provider BouncyCastleProvider)))
 
+(set! *warn-on-reflection* true)
```

Comment:
You can specify this in project.clj:

```
  :global-vars {*warn-on-reflection* true}
```

This has the added benefit that you can keep it on for just dev.